api/spec/v1.yaml

```diff
@@ -2572,6 +2596,60 @@ components:
             the `/{runtime}/accounts/{address}` endpoint.
           example: false
 
+    EvmNftList:
+      allOf:
+        - $ref: '#/components/schemas/List'
+        - type: object
+          required: [evm_nfts]
+          properties:
+            evm_nfts:
+              type: array
+              items:
+                $ref: '#/components/schemas/EvmNft'
+              description: A list of L2 EVM NFT (ERC-721, ...) instances.
+          description: A list of NFT instances.
+
+    EvmNft:
+      type: object
+      required:
+        - contract_addr
+        - id
+      properties:
+        contract_addr:
+          type: string
+          description: The Oasis address of this NFT's contract.
+          example: 'oasis1qp2hssandc7dekjdr6ygmtzt783k3gn38uupdeys'
+        eth_contract_addr:
```

Comment:
inconsistent, should be contract_eth_addr, but it's like this for consistency with EvmToken

Reply:
now uses the EvmToken struct directly